lib/providers/twitch.js

```diff
@@ -0,0 +1,27 @@
+var Hoek = require('hoek');
+var url = 'https://api.twitch.tv/kraken/';
```

Comment:
Please use the url string value in the return object below instead of a variable.